setup.py

```diff
@@ -74,7 +74,7 @@ def read(*rnames):
         'zope.schema',
         'zope.security',
         'zope.traversing',
-        ],
+    ],
```

Comment:
Let me undo this commit as it should have been done by GHA – but there seems to be a configuration problem.

Reply:
Okay, it got automatically fixed in https://github.com/zopefoundation/zope.browserpage/pull/16/commits/7adae44d3a7e69cf0b5c868d3bd1e22c688a6f0b – the missing piece was that the `pre-commit ci lite` GitHub App was only enabled for the `Zope` repository. I now enabled it for all repositories.